rtf_vis_tool/src/Components/DivGraph.js

```diff
@@ -0,0 +1,164 @@
+import React, {useEffect, useState} from "react";
+import Xarrow from "react-xarrows";
+import '../stylesheets/DivGraph.css'
+
+//Loading Components
+import Data_Association_PC from './Data_Association_PC';
+import DivNode from './DivNode';
+import EdgeList from './gtsfm_edge_list.js';
+// import FrontendSummary from './FrontendSummary';         - add this in part 3
+// import MVOSummary from './MVOSummary';                   - add this in part 3
+//import RelativeRsDivNode from './relativeRsDivNode';      - add this in part 3
+//import RotSuccessSummary from './RotSuccessSummary';      - add this in part 3
+import SfMDataDivNode from './SfMDataDivNode';
+ 
+
+//loading json files in result_metrics folders to display summary metrics
+import data_association_json from '.././result_metrics/data_association_metrics.json';
+import frontend_summary_json from '.././result_metrics/frontend_summary.json';
+import multiview_optimizer_json from '.././result_metrics/multiview_optimizer_metrics.json';
+
+//Landing Page Component to display GTSFM graph (what user first sees)
+const DivGraph = (props) => {
```

Comment:
is there any better name for this?

Reply:
sure, instead of `DivGraph.js`, I could call it `LandingPageGraph.js`, since that's more telling of the component's purpose